root/static/scripts/common/MB/Control/Menu.js

```diff
@@ -32,35 +29,31 @@ MB.Control.HeaderMenu = function () {
         return 'auto';
     }
 
-    $('.header ul.menu > li').bind('mouseenter.mb', function (event) {
-        if (self.timeout) {
-            clearTimeout(self.timeout);
-            $('.header ul.menu li ul').css('left', '-10000px');
-        }
-
-        $(this).children('ul').css('left', getLeft(this));
-    });
-
-    $('.header ul.menu li').bind('mouseleave.mb', function (event) {
-        var ul = $(this).children('ul');
-
-        self.timeout = setTimeout(function () {
-            ul.css('left', '-10000px');
-        }, self.timeout_msecs);
-    });
-
     $('.header .menu-header').on('click touchend', function (event) {
```

Comment:
I see a slight difference in behavior in iOS...with click events there's a noticeable 300 ms or so delay, and with touchend it's instant. The bootstrap menus on CB also have a delay, so it's probably advisable to remove since they're smarter than I am.